### ISSUE TITLE
Pretrained Model Incomplete

### ISSUE THREAD

**Issue by ZenithalHourlyRate** [laughtervv/3DN#4]
The pretrained model provided in the google drive only has `xxx.ckpt.meta` and `xxx.ckpt.index`, and there is no `xxx.ckpt.data-0000-of-00001` in it. Without such files, the pretrained model can not be restored.

I am here to request the releasing of these `xxx.ckpt.data-00000-of-00001`

**Comment by yifita** [comment]
I have the same loading issue.

**Comment by ghost** [comment]
Having this issue as well

**Comment by hjwdzh** [comment]
same issue. Since it has been a while, did you find the solution?

**Comment by ZenithalHourlyRate** [comment]
No, the pretrained model could only be provided by the authors, i think.

**Comment by hjwdzh** [comment]
Thanks! Seems that the data is also not provided? Or did I miss any information?

**Comment by lixryjz** [comment]
hello，is there a version of pytorch，I could‘t understand the code，are you realized it？,thanks
